File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()) return false;

        int[] charArray = new int[26];
        for(int i = 0; i < s.length();i++){
            charArray[s.charAt(i)-'a']++;
        }

        for(int i = 0; i < t.length();i++){
            charArray[t.charAt(i)-'a']--;
        }

        for(int val: charArray){
            if(val != 0) return false;
        }

        return true;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        if len(s) != len(t):
            return False
        
        count = [0] * 26
        
        for char_s, char_t in zip(s, t):
            count[ord(char_s) - ord('a')] += 1
            count[ord(char_t) - ord('a')] -= 1
        
        for i in count:
            if i != 0:
                return False
        
        return True

        